ultraplot/constructor.py

```diff
@@ -762,148 +762,187 @@ def _pop_modification(key):
     return cmap
 
 
-def Cycle(*args, N=None, samples=None, name=None, **kwargs):
-    """
-    Generate and merge `~cycler.Cycler` instances in a variety of ways.
-
-    Parameters
-    ----------
-    *args : colormap-spec or cycle-spec, optional
-        Positional arguments control the *colors* in the `~cycler.Cycler`
-        object. If zero arguments are passed, the single color ``'black'``
-        is used. If more than one argument is passed, the resulting cycles
-        are merged. Arguments are interpreted as follows:
-
-        * If a `~cycler.Cycler`, nothing more is done.
-        * If a sequence of RGB tuples or color strings, these colors are used.
-        * If a `~ultraplot.colors.DiscreteColormap`, colors from the ``colors``
-          attribute are used.
-        * If a string cycle name, that `~ultraplot.colors.DiscreteColormap`
-          is looked up and its ``colors`` are used.
-        * In all other cases, the argument is passed to `Colormap`, and
-          colors from the resulting `~ultraplot.colors.ContinuousColormap`
-          are used. See the `samples` argument.
-
-        If the last positional argument is numeric, it is used for the
-        `samples` keyword argument.
-    N
-        Shorthand for `samples`.
-    samples : float or sequence of float, optional
-        For `~ultraplot.colors.DiscreteColormap`\\ s, this is the number of
-        colors to select. For example, ``Cycle('538', 4)`` returns the first 4
-        colors of the ``'538'`` color cycle.
-        For `~ultraplot.colors.ContinuousColormap`\\ s, this is either a
-        sequence of sample coordinates used to draw colors from the colormap, or
-        an integer number of colors to draw. If the latter, the sample coordinates
-        are ``np.linspace(0, 1, samples)``. For example, ``Cycle('Reds', 5)``
-        divides the ``'Reds'`` colormap into five evenly spaced colors.
+class Cycle(cycler.Cycler):
+    def __init__(self, *args, N=None, samples=None, name=None, **kwargs):
+        """
+        Generate and merge `~cycler.Cycler` instances in a variety of ways.
```

Comment:
The docs here no longer match the fact that this is a class as opposed to a function. We should adjust.

Reply:
Good catch. I just copied the doc string over -- will fix.